config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml

```diff
@@ -234,11 +233,10 @@ periodics:
               fi
 
               set -o xtrace
-              kubetest2 tf --powervs-dns k8s-tests \
```

Comment:
I see this `--powervs-dns k8s-tests` flag is removed, its not needed anymore?

Reply:
Yes this seems to be a dummy value passed. We are not using this value anywhere in `kubetest2`.

What ever DNS information the VM/network creation uses, comes from here https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/data/powervs/main.tf#L12
The test run PASSED without this flag being passed.

@mkumatag do you have any reference on why this is needed?

Reply:
we use to use the cloud dns for name resolution, not used anymore..